config/_ytt_lib/github.com/cloudfoundry/cf-k8s-logging/config/manifests/500-log-cache-deployment.yaml

```diff
@@ -18,6 +18,37 @@ spec:
         app: log-cache
     spec:
       containers:
+      - name: cf-auth-proxy
+        image: #@ data.values.images.cf_auth_proxy
+        ports:
+          - containerPort: 8083
+        env:
+        - name: LOG_CACHE_GATEWAY_ADDR
+          value: https://localhost:8081
+        - name: CAPI_CA_PATH
+          value: /system-ca/tls.crt
+        - name: UAA_CA_PATH
+          value: /system-ca/tls.crt
+        - name: CAPI_ADDR
+          value: #@ "https://capi.{}.svc.cluster.local:80".format(data.values.system_namespace)
```

Comment:
A little confused here:
- the protocol of `CAPI_ADDR` is http*s*
- the port of `CAPI_ADDR` is 80 — usually reserved for plaintext HTTP
- looking at the API server, the `capi` Service names port 80 as http.  We assume the API server is actually serving plaintext on `:80` (https://github.com/cloudfoundry/capi-k8s-release/blob/98b34698e61332be8fa7955d793cd60880208ed2/templates/service.yml#L10-L13)
- We're talkin' plaintext to UAA, but TLS to CAPI?!

Could you help me understand?

Reply:
We tried using https for both, but uaa didn't accept https. We saw capi_addr was 80 on the service, so we used that. https://github.com/cloudfoundry/cf-for-k8s/blob/409efb3bbe6df6bd6e53ee71aa2ad9fac30e5b0c/config/_ytt_lib/github.com/cloudfoundry/capi-k8s-release/templates/service.yml#L10. I'm not sure why capi actually accepts(or seems to accept https), but it does.

Reply:
Got it... thank you for that explanation, @Benjamintf1.

I'm parsing your comment precisely and reading that CAPI might well serve HTTP, but y'all hadn't tried (expecting, at this time, to be trying to make this communication encrypted)... is that right?

Either way, we've been assured by CF for K8s Networking that Istio has been configured to encrypt traffic between services.  You should be able to serve and consume plaintext.

Reply:
yup, we hadn't attempted that yet.

Reply:
We did it